config/application.rb

```diff
@@ -33,8 +33,10 @@ class Application < Rails::Application
     config.watchable_dirs['lib'] = [:rb]
 
     # TODO(#45): add rack-cors middleware to streamline CORS config
+    # TODO(crew): add a variable for origin domain.
     config.action_dispatch.default_headers = {
-      'Access-Control-Allow-Headers' => 'Authorization'
+      'Access-Control-Allow-Headers' => 'Authorization',
+      'Access-Control-Allow-Origin' => 'http://localhost:4000'
```

Comment:
This should be moved to an environment level config, instead of in the global application config.